### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by ekohl**
See individual commits

### CODE REVIEW

File: lib/tool_belt/issue_service.rb [hunk 1]
```diff
@@ -3,28 +3,30 @@
...
 
-    def release_issues(release_id)
-      project.get_issues_for_release(release_id)
+    def release_issues(version_id)
+      project.issues_for_version(version_id)
     end
 
-    def load_issues
-      issues = release_issues(redmine_release_id)
+    def load_issues(include_prior)
```
Comment (ehelms) [inline lib/tool_belt/issue_service.rb]: Can we set this to false by default so functions don't have to pass this in?

**Comment by ehelms** [comment]
Can you paste a sample generated changelog snippet?

File: lib/tool_belt/issue_service.rb (hunk 1 above)
Comment (ekohl) [reply lib/tool_belt/issue_service.rb]: I intended to do that, but clearly didn't.

**Comment by ekohl** [comment]
> Can you paste a sample generated changelog snippet?

https://gist.github.com/ekohl/fd786adae13051856927fd2697cfea90 was generated using this code

**Comment by ekohl** [comment]
I do see that the generated github URLs are just wrong if redmine identifier != github identifier but I'm pretty certain that was already a problem in the existing code.

File: lib/tool_belt/issue_service.rb (hunk 1 above)
Comment (ekohl) [reply lib/tool_belt/issue_service.rb]: Question is if it should default to true or false. True would be backwards compatible but a user might not expect this and false might make more sense.

Comment (ehelms) [reply lib/tool_belt/issue_service.rb]: We also now have the multiple releases way of saying an issue was delivered which simplify/complicates things as well.

**Pull request merged**